src/lib/openapi/spec/event-schema.ts

```diff
@@ -0,0 +1,53 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { tagSchema } from './tag-schema';
+
+export const eventSchema = {
+    $id: '#/components/schemas/eventSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['id', 'createdAt', 'type', 'createdBy'],
+    properties: {
+        id: {
+            type: 'integer',
+            minimum: 1,
```

Comment:
I wonder if we need to set minimum for `id`s. The user should only have to worry about the type, at most.

Reply:
Any reason why we wouldn't? The minimum _does_ provide extra type information (for languages with natural numbers, non-zero ints, dependent types, etc).

Reply:
If it was a string ID I would be careful locking down its format (should be opaque). Probably not important here.